cmd/kyverno/main.go

```diff
@@ -73,6 +74,7 @@ func main() {
 	flag.StringVar(&kubeconfig, "kubeconfig", "", "Path to a kubeconfig. Only required if out-of-cluster.")
 	flag.StringVar(&serverIP, "serverIP", "", "IP address where Kyverno controller runs. Only required if out-of-cluster.")
 	flag.BoolVar(&profile, "profile", false, "Set this flag to 'true', to enable profiling.")
+	flag.BoolVar(&generateSuccessEvents, "generateSuccessEvents", false, "Set this flag to 'true', to generate success events.")
```

Comment:
Do we want to configure it via the flag? We can default the value in the code, and let the user configure it via the ConfigMap, which can take the value dynamically.

Do you see any other use case that we want to restart the Pod when the value is updated?